### PR TITLE
Updated description field of OLM metadata

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

**Review by jwendell — APPROVED** [review]
LGTM, only a couple of minor questions:

1. Are the descriptions going to be the same for maistra and product? I mean, maistra's description is 'OpenShift Service Mesh'.

1. Do you need a Jira for this?

**Comment by dmitri-d** [comment]
> Are the descriptions going to be the same for maistra and product? I mean, maistra's description is 'OpenShift Service Mesh'.

Not sure at this point, figured I'll sort out the servicemesh stuff first.

**Comment by dmitri-d** [comment]
* Added ticket number

File: manifests-maistra/1.0.0/maistra.v1.0.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -9,10 +9,10 @@ metadata:
     certified: "false"
     repository: https://github.com/maistra/istio-operator
     description: |-
-      The Service Mesh Operator provides means for installing, configuring, and managing an instance of Red Hat OpenShift Service Mesh.
+      The OpenShift Service Mesh Operator enables you to install, configure, and manage an instance of Red Hat OpenShift Service Mesh.
```
Comment (brian-avery) [inline manifests-maistra/1.0.0/maistra.v1.0.0.clusterserviceversion.yaml:12]: Should this be labeled as Maistra rather than OpenShift Service Mesh?

**Review by brian-avery — APPROVED** [review]

File: manifests-maistra/1.0.0/maistra.v1.0.0.clusterserviceversion.yaml (hunk 1 above)
Comment (brian-avery) [reply manifests-maistra/1.0.0/maistra.v1.0.0.clusterserviceversion.yaml:12]: I see that Jonh asked this already. I'm good with it for now. I'm guessing that eventually we will want to differentiate the two though.

**Comment by jwendell** [comment]
@dmitri-d conflict, I think it's just the dates, but I'll leave that for you to take a look.

**Comment by dmitri-d** [comment]
* rebased

**Pull request merged**